docs/developer-tutorials/local-test-setup.md

```diff
@@ -88,8 +91,14 @@ This setup is deployed with the following security features enabled:
    Adjust the path to your `template-operator` local directory or any other reference module operator accordingly.
 
    ```shell
-   kyma alpha create module -p ../template-operator --version 1.2.3 \
-   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
+   cd <local path to template operator repository>
+   
+   # generates the manifests and saves them to a template-operator.yaml file
+   make build-manifests
+   
+   # create the a ModuleTemplate CR and saves it to a template-yaml file
```

Comment:
```suggestion
   # create a ModuleTemplate CR and save it to a template YAML file
```
In the previous set of comments, you used the second-person singular (user perspective), now third-person singular (command perspective), let's unify.